### PR TITLE
Fix Sorting Algorithm Bug

### LINKED ISSUE #12: IP Address Sorting Result Bug
**Describe the bug**
I got an issue on your subping IP Address Result that not sorted in a correct way as shown below :

image

**Expected behavior**
It Should be appeard in a sequence of IP Address based on the IP Ranges.

please notice this if you find the same issue as mine

### PR DESCRIPTION
**Pull request by izzatbey**
Solution of the Sorting Algorithm bug in subping result

### CODE REVIEW

File: cmd/subping/main.go [hunk 1]
```diff
@@ -64,33 +66,37 @@ func main() {
 	fmt.Printf("Network\t\t: %s\n", subnetString)
 	fmt.Printf("IP Ranges\t: %v - %v\n", ips[0], ips[len(ips)-1])
 	fmt.Printf("Total hosts\t: %d\n", len(ips))
-	fmt.Println("---------------------------------------")
-	fmt.Println("| IP Address       | Avg Latency      |")
-	fmt.Println("---------------------------------------")
+	fmt.Println("-------------------------------------")
+	fmt.Println("| IP Address      | Avg Latency     |")
+	fmt.Println("-------------------------------------")
```
Comment (fadhilyori) [inline cmd/subping/main.go]: Why did you reduce the length of the table?

File: cmd/subping/main.go [hunk 2]
```diff
@@ -64,33 +66,37 @@ func main() {
...
 	fmt.Print("\r")
 
 	for _, ip := range keys {
-		stats := results[ip]
+		// convert bytes to string in each line of IP
+		ipString := ip.String()
+		stats := results[ipString]
 
-		fmt.Printf("| %-16s | %-16s |\n", ip, stats.AvgRtt.String())
+		fmt.Printf("| %-15s | %-15s |\n", ipString, stats.AvgRtt.String())
```
Comment (fadhilyori) [inline cmd/subping/main.go]: Why did you reduce the length of the table?

File: cmd/subping/main.go [hunk 3]
```diff
@@ -64,33 +66,37 @@ func main() {
...
 	}
 
-	fmt.Println("---------------------------------------")
+	fmt.Println("-------------------------------------")
```
Comment (fadhilyori) [inline cmd/subping/main.go]: Why did you reduce the length of the table?

**Review by fadhilyori — CHANGES_REQUESTED** [review]

File: cmd/subping/main.go (hunk 2 above)
Comment (izzatbey) [reply cmd/subping/main.go]: sorry i forgot to change back... it just myself understanding the table... i'll bring it back

**Review by fadhilyori — APPROVED** [review]
lgtm

**Review by fadhilyori — APPROVED** [review]
lgtm

**Pull request merged**